### PR TITLE
gen_sqw bug

### PR DESCRIPTION
**Pull request by abuts**
This is partial fix of the issue, reported at #83 as more changes have to be done in Horace (which may request relevant changes in Herbert)
Here I am enabling better error reporting from failing MPI job and fixing serialization on rundata(h) class (partially in Horace and some unit tests for the change are also in Horace) 

More changes may be necessary

### CODE REVIEW

**Comment by hsautessella** [comment]
Failures on Windows in this PR look as if they're due to #82

**Comment by abuts** [comment]
@hsautessella  the first failure was caused by incorrect line ending settings when Jenkins was pulling the git, the second -- not so sure -- I believe it was fixed and certainly not repeated.

File: _test/test_data_loaders/test_rundata.m [hunk 1]
```diff
@@ -345,6 +347,33 @@ function test_serialization_crystal(this)
...
+        function test_serialize_on_file(obj)
+            %
+            run=rundata(f_name(obj,'MAP11014.nxspe'));
+            db = run.serialize();
+            runr = rundata.deserialize(db);
+            
+            assertEqual(run,runr);
+        end
+        %
+        function test_serialize_in_memory(obj)
```
Comment (nickbattam-tessella) [inline _test/test_data_loaders/test_rundata.m]: what is this test testing? The previous test has shown `run === runr`, this seems to test that `run === runr` with some fields removed

File: herbert_core/classes/data_loaders/@asciipar_loader/asciipar_loader.m [hunk 2]
```diff
@@ -187,6 +187,12 @@
                 return
             end
             [this.det_par_,this.n_detinpar_,this.par_file_name_] = check_det_par(value);
+            % HACK:
```
Comment (nickbattam-tessella) [inline herbert_core/classes/data_loaders/@asciipar_loader/asciipar_loader.m]: what problem is this `HACK` solving?

File: _test/test_data_loaders/test_rundata.m (hunk 1 above)
Comment (abuts) [reply _test/test_data_loaders/test_rundata.m]: it checks that serialization/deserialization work

File: herbert_core/classes/data_loaders/@asciipar_loader/asciipar_loader.m (hunk 2 above)
Comment (abuts) [reply herbert_core/classes/data_loaders/@asciipar_loader/asciipar_loader.m]: the next commit rectifies this.

**Comment by nickbattam-tessella** [comment]
SL7-2017b build failure is expected; that build configuration is not currently working

**Pull request merged**